stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
I agree that maintaining a design system is better, but this project is so big, there is such a big mess everywhere, that I don't think that another abstraction will improve. 

I also don't have enough information to find a decent, future proof name. The redesign is happening in multiple phases and it's not clear what this modifier will represent.

And there have been so many redesigns that I don't think the design system is clear at the moment.

With all of this, this color name makes it easy to know what the modifier does and easy to change in the future.